README.md

```diff
@@ -1,12 +1,35 @@
 # Bazel Dev Container Feature
 
-💚 Installs Bazelisk and Buildifier
+💚 Installs Bazel and some extra tools
 
 <p align=center>
   <img width=400 src="https://github.com/devcontainers-community/features-bazel/assets/61068799/48f7037a-f389-463c-b9b5-954f169f8aaf">
 </p>
 
-Install [Bazelisk](https://github.com/bazelbuild/bazelisk) and [Buildifier](https://github.com/bazelbuild/buildtools/blob/master/buildifier/README.md).
+> _{Fast, Correct} - Choose two_
+>
+> Build and test software of any size, quickly and reliably.
+>
+> - **Speed up your builds and tests**: Bazel rebuilds only what is necessary.
+>   With advanced local and distributed caching, optimized dependency analysis
+>   and parallel execution, you get fast and incremental builds.
+> - **One tool, multiple languages**: Build and test Java, C++, Android, iOS,
+>   Go, and a wide variety of other language platforms. Bazel runs on Windows,
+>   macOS, and Linux.
+> - **Scalable**: Bazel helps you scale your organization, codebase, and
+>   continuous integration solution. It handles codebases of any size, in
+>   multiple repositories or a huge monorepo.
+> - **Extensible to your needs**: Easily add support for new languages and
+>   platforms with Bazel's familiar extension language. Share and re-use
+>   language rules written by the growing Bazel community.
+
+<!-- prettier-ignore -->
+&mdash; [bazelbuild/bazel: a fast, scalable, multi-language and extensible build system](https://github.com/bazelbuild/bazel)
+
+This Feature installs Bazel using [Bazelisk]. Then, it symlinks
+`/usr/local/bin/bazel` to the Bazelisk installation so that you can use `bazel`
+as normal through `bazelisk`. It also installs the [Bazel VS Code extension].
```

Comment:
The main feature of Bazelisk is that it automatically uses the correct Bazel version based on your environment (file, env var, ...).